blobs.go

```diff
@@ -75,6 +75,11 @@ type BlobDeleter interface {
 	Delete(ctx context.Context, dgst digest.Digest) error
 }
 
+// BlobEnumerator allows to list blobs in storage.
+type BlobEnumerator interface {
+	Enumerate(ctx context.Context) ([]digest.Digest, error)
```

Comment:
This allocates way too much memory to serve large requests. Please follow the approach done for `Namespace.Repositories` so that the dataset can be served in a blocks.